miden-lib/asm/miden/kernels/tx/prologue.masm

```diff
@@ -790,43 +787,34 @@ end
 
 #! Process the input notes data provided via the advice provider. This involves reading the data
 #! from the advice provider and storing it at the appropriate memory addresses. As each note is
-#! processed its hash and nullifier is computed. The transaction nullifier commitment is computed
-#! via a sequential hash of all (nullifier, ZERO) pairs for all input notes.
+#! processed its hash and nullifier are computed. The transaction nullifier commitment is computed via
+#! a sequential hash of all (nullifier, ZERO) pairs for all input notes.
 #!
 #! Stack: []
-#! Advice stack: [num_cn, ...],
-#! Advice map: {
-#!               NC: [
-#!                  CN1_SN, CN1_SR, CN1_IR, CN1_VR, CN1_M, CN1_NA
-#!                  CN1_A1, CN1_A2, ...
-#!
-#!                  CN2_SN,CN2_SR, CN2_IR, CN2_VR, CN2_M, CN2_NA
-#!                  CN2_A1, CN2_A2, ...
-#!                  ...]
-#!              }
+#! Advice stack: [num_notes],
+#! Advice map: { NC => [(SN, SR, IR, VR, M, NA, A0, ..., An){num_notes}] }
 #! Output: []
 #!
-#! - num_cn is the number of input notes.
-#! - NC is the input note nullifier commitment.
-#! - CN1_SN is the serial number of input note 1.
-#! - CN1_SR is the script root of input note 1.
-#! - CN1_IR is the inputs root of input note 1.
-#! - CN1_VR is the vault root of input note 1.
-#! - CN1_M is the metadata of input note 1.
-#! - CN1_NA are optional note args for input note 1.
-#! - CN1_A1 is the first asset of input note 1.
-#! - CN1_A2 is the second asset of input note 1.
+#! - num_notes is the number of input notes.
+#! - NC is the nullifier commitment of all input notes.
+#! - SN is the serial number of the nth input note.
+#! - SR is the script root of the nth input note.
+#! - IR is the inputs root of the nth input note.
+#! - VR is the vault root of the nth input note.
+#! - M is the metadata of the nth input note.
+#! - NA are optional note args for the nth input note.
+#! - A0 is the first asset of the nth input note.
+#! - An is the second asset of the nth input note.
```

Comment:
Seems like this would be the nth asset instead of the second asset. Additionally `n` here is representing two different things (nth input note and nth asset). Maybe this should be expressed as it is on `process_input_note` (`A0..An are..`).

Reply:
Good point, changed